packages/block-editor/src/components/block-preview/index.js

```diff
@@ -24,10 +24,15 @@ export function BlockPreview( {
 	__experimentalLive = false,
 	__experimentalOnClick,
 } ) {
-	const settings = useSelect(
+	const originalSettings = useSelect(
 		( select ) => select( blockEditorStore ).getSettings(),
 		[]
 	);
+	const settings = useMemo( () => {
+		const _settings = { ...originalSettings };
+		_settings.__experimentalBlockPatterns = [];
+		return _settings;
+	}, [ originalSettings ] );
```

Comment:
What's the reasoning behind this change?

Reply:
We don't want to provide patterns in block previews. Since we use the `usePreParsePatterns` in the `BlockList` component, it would start parsing patterns for every block preview.